packages/components/src/components/toast/toast-item/toast-item.tsx

```diff
@@ -56,15 +63,15 @@ export class ToastItem {
     const PrefixedTagNames = getPrefixedTagNames(this.host);
 
     return (
-      <Host>
+      <Host popover="manual">
         <PrefixedTagNames.pIcon
           class="icon"
           name={getInlineNotificationIconName(this.state)}
           color={`notification-${this.state}` as IconColor}
           theme={this.theme}
           aria-hidden="true"
         />
-        <p id={toastId} class="content" role="status" aria-live="polite" innerHTML={this.text}></p>
+        <p id={toastId} class="content" innerHTML={this.text}></p>
```

Comment:
Do we still need class="content"? It might be also styled directly on p.

Reply:
fixed